create-proxy.sh

```diff
@@ -56,7 +56,11 @@ echo "Creating container [${NAME}] using image [${CONTAINER_IMAGE}:${CONTAINER_V
 
 COMMAND=""
 
-COMMAND="deluser zabbix; addgroup -g ${group} zabbix; adduser -u ${user} -G zabbix -g 'zabbix user' -s /sbin/nologin -D zabbix; find / -user 100 -exec chown zabbix {} \; ; find / -group 1000 -exec chgrp zabbix {} \; ; "
+if ["${CONTAINER_VERSION}" == "ubuntu-4.0-latest"]; then
```

Comment:
I suggest that the comparison is not fixed to a specific ubuntu version. Instead it could test if "ubuntu" string is found in the variable.

Reply:
I agree.
`if [[ "${CONTAINER_VERSION}" =~ ^ubuntu ]]; then`